packages/ibm-products-styles/src/components/NotificationsPanel/_notifications-panel.scss

```diff
@@ -179,8 +179,9 @@ $block-class: #{c4p-settings.$pkg-prefix}--notifications-panel;
       .#{$block-class}__notification-read-less-button {
         @include type.type-style('label-01');
 
+        display: flex;
+        align-items: center;
```

Comment:
Fixes styling for "Read more"

**Before**
![image](https://github.com/user-attachments/assets/6e61b51c-6b02-41de-bd65-ab0d2a78146f)

**After**
![image](https://github.com/user-attachments/assets/2da118e1-6ac7-4388-bb27-86daba654811)